packages/copyable/src/Copyable/Copyable.tsx

```diff
@@ -85,6 +86,14 @@ export default function Copyable({
       container: portalContainer,
     });
 
+    clipboard.on('success', (event: React.ClipboardEvent<HTMLDivElement>) => {
+      onCopy?.(event);
+    });
+
+    clipboard.on('error', (event: React.ClipboardEvent<HTMLDivElement>) => {
+      onCopy?.(event);
+    });
```

Comment:
at this point, is there a reason for doing it this way vs calling `onCopy` in the `onClick` handler?

Reply:
so we can pass the consumer the same event information that we get ... the onClick won't have the text/trigger args

Reply:
now we're passing the clipboard event not the onClick event